taas/requirements.in

```diff
@@ -0,0 +1,13 @@
+behave
+connexion
+# behave-pytest # Disabled due to https://github.com/ribozz/behave-pytest/issues/2
+pytest
+jsonschema
+requests
+docker-py
+pyjwt
+pycrypto
```

Comment:
Is this the dependency with several known vulnerabilities? :)

Reply:
Oh yes :) I'd need to wait for the `fabric8-analytics-auth` implementation and then leverage it.